### PR TITLE
Updated reproduction log

### PR DESCRIPTION
**Pull request by baixabhi**
Updated the evaluation commands for various conditions and topics related to MS MARCO V1 Passage. Renamed the msmarco-v2.1-doc.yaml file to msmarco-v2.1.yaml. Also updated the metric evaluation output formatting.

### CODE REVIEW

**Comment by baixabhi** [comment]
Kindly review my pr. Thank you

**Comment by lintool** [comment]
Please fix conflicts?

**Comment by baixabhi** [comment]
Done

File: docs/start-here.md [hunk 1]
```diff
@@ -379,7 +379,10 @@ If you think this guide can be improved in any way (e.g., you caught a typo or t
 + Results reproduced by [@a68lin](https://github.com/a68lin) on 2024-04-11 (commit [`39cecf6`](https://github.com/castorini/anserini/commit/39cecf6c257bae85f4e9f6ab02e0be101338c3cc))
 + Results reproduced by [@DanielKohn1208](https://github.com/DanielKohn1208/) on 2024-04-21 (commit [`9863611d`](https://github.com/castorini/anserini/commit/9863611d307773c086e64496a2a94cf6599c28b0))
 + Results reproduced by [@emadahmed19](https://github.com/emadahmed19) on 2024-04-28 (commit [`a4064a6`](https://github.com/castorini/anserini/commit/a4064a6fcf6adc7a2cdb5f94e2959f6e3904d916))
+
```
Comment (lintool) [inline docs/start-here.md:382]: remove extra line

File: docs/experiments-msmarco-passage.md [hunk 2]
```diff
@@ -482,8 +482,10 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@a68lin](https://github.com/a68lin) on 2024-04-11 (commit [`39cecf6`](https://github.com/castorini/anserini/commit/39cecf6c257bae85f4e9f6ab02e0be101338c3cc))
 + Results reproduced by [@DanielKohn1208](https://github.com/DanielKohn1208/) on 2024-04-21 (commit [`9863611d`](https://github.com/castorini/anserini/commit/9863611d307773c086e64496a2a94cf6599c28b0))
 + Results reproduced by [@emadahmed19](https://github.com/emadahmed19) on 2024-04-28 (commit [`a4064a6`](https://github.com/castorini/anserini/commit/a4064a6fcf6adc7a2cdb5f94e2959f6e3904d916))
++ Results reproduced by [@baixabhi](https://github.com/baixabhi) on 2024-05-09 (commit [`8e188670`](https://github.com/castorini/anserini/commit/8e188670e038c79782302ee3596828ea688250e0))
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:485]: please put in chronological order.

**Review by baixabhi — COMMENTED** [review]
Done

**Review by lintool — APPROVED** [review]

**Pull request merged**